src/vs/workbench/contrib/preferences/browser/settingsWidgets.ts

```diff
@@ -1150,20 +1155,25 @@ export class ObjectSettingDropdownWidget extends AbstractListSettingWidget<IObje
 		return true;
 	}
 
-	protected getLocalizedRowTitle(item: IObjectDataItem): string {
-		let enumDescription = item.key.type === 'enum'
-			? item.key.options.find(({ value }) => item.key.data === value)?.description
-			: undefined;
+	protected addTooltipsToRow(rowElement: HTMLElement, item: IObjectDataItem): void {
+		const keyElement = rowElement.getElementsByClassName('setting-list-object-key')[0] as HTMLElement;
```

Comment:
Avoid querying for elements like this. Set these attributes when creating these elements. Or if you can't do that, you could make each row a class, or use a template-like object that has property references to elements that are needed externally